assets/device/classes/airconditioning.json

```
@@ -0,0 +1,8 @@
+{
+  "title": {
+    "en": "Airconditioning"
+  },
+  "description": {
+    "en": "Use this device class for airconditioners, either mobile or split type units."
```

```suggestion
    "en": "Use this device class for airconditioners, either portable or split type units."
```

Portable seems to be the correct term: https://www.coolblue.nl/en/air-conditioners/portable-air-conditioners